src/FormTaxonomies/Actions/EnqueueFormBuilderAssets.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace Give\FormTaxonomies\Actions;
```

Comment:
@kjohnson what's your reasoning behind making FormTaxonomies a top level domain in Give? I would think a lot of this should be within the DonationForms and/or FormBuilder domains.

Reply:
Form taxonomies are an optional feature, so I think it makes sense to treat it almost like a built-in add-on.

Also, taxonomies are a WordPress concept and do not _belong_ to the donation form domain.

Reply:
@kjohnson thanks for your insight.  Sounds like your viewing this as a separation of concerns, which I agree from a WordPress concept technically makes sense.  

I'm going to think out loud here for a minute and would appreciate your thoughts to help come to a conclusion 😄 

When it comes to domains, I always want to think about the top level business logic and context, this is in contrast to a structure of types.  What encompasses a donation form?  Well, we have things like form settings, form schema, front-end rendering, form request, server side validation, persistence, and even the form confirmation.  How about the form builder? well we have blocks, rendering, form request, persistence, etc.

Where do taxonomies fit in? Well, to your point, they are a separate WordPress concept that is used for content organization and relationships.  Taxonomies are also both a concept and a type at the same time (right?). However, our _usage_ of these taxonomies is very contextual in that they are used within donation forms.  So we are essentially utilizing a WordPress concept in a domain-contextual way (What's a form taxonomy without a donation form?).

Looking at our current structure and also our future, I think this comes down to a two-part questions in my mind:
If we were going with FormTaxonomies as a top level concept, would we potentially have more top level taxonomy domains like CampaignTaxonomies, EventTaxonomies, etc.?  Or would the contextual usage of those warrant them to live inside their respected domains?

Reply:
> [o]ur usage of these taxonomies is very contextual in that they are used within donation forms.

These custom taxonomies are not used exclusively _within_ donation forms. Rather, they are managed at a global level in the WordPress admin, they are managed per-form in the Form Builder, and they are used as input for various shortcodes (such as the Form Grid) for querying donations forms in the database.

They aren't actually used by the donation form at all.

Reply:
> If we were going with FormTaxonomies as a top level concept, would we potentially have more top level taxonomy domains like CampaignTaxonomies, EventTaxonomies, etc.? Or would the contextual usage of those warrant them to live inside their respected domains?

I could see those being additional top level domain.

I would say, "it depends".

Reply:
Here’s my 2c: I don’t think a custom taxonomy should be a top-level domain. It seems more appropriate for it to be closely attached to the DonationForm domain, similar to its own post type. A taxonomy is rarely an independent entity, it is usually linked to a specific post type. In our case, it should be related to the donation form post type. Additionally, the interface for managing the new taxonomy in the VFB should live within the same area as the other form settings.

Reply:
Thanks for the discussion @kjohnson and @pauloiankoski .  I'll leave the final decision up to you @kjohnson, just wanted to make sure we thought through this as we continue to build out our functionality.

If you decide to keep this separated domain than I left some suggestions on handling this as an extension.